packages/core/helper-plugin/lib/src/components/ConfirmDialog/index.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { Dialog, DialogBody, DialogFooter } from '@strapi/parts/Dialog';
+import { Stack } from '@strapi/parts/Stack';
+import { Row } from '@strapi/parts/Row';
+import { Text } from '@strapi/parts/Text';
+import { Button } from '@strapi/parts/Button';
+import AlertWarningIcon from '@strapi/icons/AlertWarningIcon';
+import DeleteIcon from '@strapi/icons/DeleteIcon';
+
+const ConfirmDialog = ({ isConfirmButtonLoading, onToggleDialog, onConfirm, isVisible }) => {
+  const { formatMessage } = useIntl();
+
+  if (!isVisible) {
+    return null;
+  }
+
+  return (
+    <Dialog
+      onClose={onToggleDialog}
+      title={formatMessage({
+        id: 'Settings.webhooks.confirmation',
+        defaultMessage: 'Confirmation',
+      })}
+      labelledBy="confirmation"
+      describedBy="confirm-description"
+    >
+      <DialogBody icon={<AlertWarningIcon />}>
+        <Stack size={2}>
+          <Row justifyContent="center">
+            <Text id="confirm-description">
+              {formatMessage({
+                id: 'Settings.webhooks.confirmation.delete',
```

Comment:
You can use this translation here:
```
components.popUpWarning.message
```
Similarly to the title can you make it a props?